### DISCUSSION TITLE
Missed recording

### DISCUSSION THREAD

**Discussion by blacknell** [blakeblackshear/frigate#12687] [General]
The other morning I woke up to find some prankster had put a road sign trestle on our front drive during the night.

![snapshot](https://github.com/user-attachments/assets/0d6128a4-1392-49ca-8d11-dd65f6155ff3)

However, frigate didn't record it. The snapshot shows someone at 01:37:51 but the recording clip starts at 01:37:56

It's never done this before to my recollection. I was able to scroll recordings and I could see it skip over the relevant seconds but now I can't find how to repeat that. All I can now do is scroll through recordings and watch the one that starts at 01:37:52 (different start time to the event clip above)

My config is retain motion for 2 days, active objects events for 22 days (persons) and pre/post capture for 5s.

Any ideas what has happened?

**Comment by NickM-27** [comment]
It's possible the recording and detect streams were out of sync.

**Reply by blacknell** [reply]
By 5s? Would increasing pre/post capture reduce the risk of that happening?

**Reply by NickM-27** [reply]
I've seen it up to 30 seconds before

Maybe, it would be easier to debug in 0.14 with the previews. Could also be something else in the config. I'd suggest comparing the time in the snapshot, time in the UI, and time that the icons above the clip show

**Reply by blacknell** [reply]
I've clicked star on the event so hopefully this retained and when I upgrade my prod to 0.14 I can have a look. In 0.13 the times are as above

**Reply by NickM-27** [reply]
I meant if it happened while running 0.14
